app/assets/styles/admin.scss

```diff
@@ -0,0 +1,15 @@
+// About page 
+.updated_alert, .ui.button.cancel-edit, .ui.button.end-edit {
+	display: none;

```

Comment:
In general for scss, delete one level of indent than it is here for the properties
(similar indentation to how the corresponding css file is below)